jr-stree/src/main/java/com/fasterxml/jackson/jr/stree/JacksonJrsTreeCodec.java

```diff
@@ -32,49 +37,53 @@ public <T extends TreeNode> T readTree(JsonParser p) throws IOException {
         return (T) nodeFrom(p);
     }
 
-    private JrsValue nodeFrom(JsonParser p) throws IOException
-    {
+    private JrsValue nodeFrom(JsonParser p) throws IOException {
         int tokenId = p.hasCurrentToken()
                 ? p.currentTokenId() : p.nextToken().id();
-        
+
         switch (tokenId) {
-        case JsonTokenId.ID_TRUE:
-            return JrsBoolean.TRUE;
-        case JsonTokenId.ID_FALSE:
-            return JrsBoolean.FALSE;
-        case JsonTokenId.ID_NUMBER_INT:
-        case JsonTokenId.ID_NUMBER_FLOAT:
-            return new JrsNumber(p.getNumberValue());
-        case JsonTokenId.ID_STRING:
-            return new JrsString(p.getText());
-        case JsonTokenId.ID_START_ARRAY:
-            {
+            case JsonTokenId.ID_TRUE:
+                return JrsBoolean.TRUE;
+            case JsonTokenId.ID_FALSE:
+                return JrsBoolean.FALSE;
+            case JsonTokenId.ID_NUMBER_INT:
+            case JsonTokenId.ID_NUMBER_FLOAT:
+                return new JrsNumber(p.getNumberValue());
+            case JsonTokenId.ID_STRING:
+                return new JrsString(p.getText());
+            case JsonTokenId.ID_START_ARRAY: {
                 List<JrsValue> values = _list();
                 while (p.nextToken() != JsonToken.END_ARRAY) {
                     values.add(nodeFrom(p));
                 }
                 return new JrsArray(values);
             }
-        case JsonTokenId.ID_START_OBJECT:
-            {
+            case JsonTokenId.ID_START_OBJECT: {
```

Comment:
These style changes make it very difficult to review actual changes. Any chance they could be avoided?

Reply:
understood, will avoid this!